src/maker/config.rs

```diff
@@ -45,32 +45,39 @@ pub struct MakerConfig {
     /// This field will be removed in a future version as the application will be Tor-only.
     /// Clearnet support is being phased out for security reasons.
     pub connection_type: ConnectionType,
+    /// A fixed base fee charged by the Maker for providing its services
+    pub base_fee: u64,
+    /// A percentage fee based on the swap amount.
+    pub amount_relative_fee_pct: f64,
```

Comment:
These are not being written in the in the config file. The `write_to_file` API needs to be updated with these values. The comment above the values should also be written to the file.